controllers/antreainstall_controller.go

```diff
@@ -366,14 +367,26 @@ func (r *AntreaInstallReconciler) getAppliedOperConfig() (*operatorv1.AntreaInst
 		return r.AppliedOperConfig, nil
 	}
 	operConfig := &operatorv1.AntreaInstall{}
-	antreaConfig := corev1.ConfigMap{}
-	if err := r.Client.Get(context.TODO(), types.NamespacedName{Namespace: operatortypes.AntreaNamespace, Name: operatortypes.AntreaConfigMapName}, &antreaConfig); err != nil {
```

Comment:
Antrea configmap name is auto-generated with suffix, so cannot get configmap with name.

[https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/antrea-manifest/antrea.yml#L1313](https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/antrea-manifest/antrea.yml#L1313)